src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -0,0 +1,112 @@
+package org.rust.ide.core.overrideImplement
```

Comment:
[one](https://github.com/JetBrains/kotlin/tree/master/idea/src/org/jetbrains/kotlin/idea/util), [two](https://github.com/JetBrains/kotlin/tree/master/idea/idea-core/src/org/jetbrains/kotlin/idea/core), [three](https://github.com/JetBrains/kotlin/tree/master/idea/ide-common/src/org/jetbrains/kotlin/util)
Looks like a good practice to distinguish Kotlin files from Kotlin classes.